docs/pages/documentation/getting-started/index.md

```diff
@@ -31,6 +31,6 @@ fvm flutter doctor
 
 ## Next Steps
 
-1. [Install FVM](./installation) on your system
-2. [Configure](./configuration) your first project
-3. Check the [FAQ](./faq) for common questions
+1. [Install FVM](../getting-started/installation) on your system
```

Comment:
I think this needs to be ./ instead of ../ as you can see on the preview it does not seem to be working correctly
https://fvm-git-fork-moritzmessner-fix-typos-fluttertools.vercel.app/

Reply:
@leoafarias thanks for the feedback, i adjusted the path, kindly check again